arrow/parser.py

```diff
@@ -387,12 +376,14 @@ def _parse_token(self, token, value, parts):
             parts["weekdate"] = value
 
     @staticmethod
-    def _build_datetime(parts):
+    def _build_datetime(parts: Dict[str, Any]) -> datetime:
 
         weekdate = parts.get("weekdate")
 
         if weekdate is not None:
             # we can use strptime (%G, %V, %u) in python 3.6 but these tokens aren't available before that
```

Comment:
We can make this internal change for `1.0.0` or leave it until afterwards.

Reply:
want to make an issue @systemcatch ?